utils/forest_shim/src/sector.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+pub use fvm_shared::sector::RegisteredSealProof as RegisteredSealProofV2;
+pub use fvm_shared::sector::SectorInfo as SectorInfoV2;
+pub use fvm_shared3::sector::RegisteredSealProof as RegisteredSealProofV3;
+pub use fvm_shared3::sector::SectorInfo as SectorInfoV3;
+use fvm_shared3::sector::SectorNumber;
+use std::ops::Deref;
+
+/// Represents a shim over RegisteredSealProof from fvm_shared with convenience methods to convert to an older version of the type
+pub struct RegisteredSealProof(pub RegisteredSealProofV3);
```

Comment:
Instead of making it public, we could implement `From<RegisteredSealProofV3>`. It would nicely encapsulate it.

Reply:
you're talking about the inner pub?